src/dataform.js

```diff
@@ -0,0 +1,61 @@
+const { DataformClient } = require('@google-cloud/dataform').v1beta1;
+
+const dataformClient = new DataformClient();
+
+/**
+ * Get Dataform compilation result.
+ *
+ * @param {string} repoURI Dataform repository URI.
+ * @returns {object} Compilation result.
+ */
+async function get_compilation_results(repoURI) {
+  const request = {
+    parent: repoURI,
+    compilationResult: {
+      gitCommitish: 'main'
+    }
+  }, dev_request = {
+    parent: repoURI,
+    compilationResult: {
+      gitCommitish: 'dev'
+    },
+    codeCompilationConfig: {
+      schemaSuffix: 'dev',
+      tablePrefix: 'dev',
+      vars: {
+        current_month: '2024-08-01',
+      },
+    }
+  };
+
+  const [response] = await dataformClient.createCompilationResult(dev_request);
```

Comment:
To adjust before merging:
```suggestion
  const [response] = await dataformClient.createCompilationResult(request);
```